### PR TITLE
Monocle operator image

### PR DESCRIPTION
**Pull request by fserucas**
This Change introduces the following changes:

Adds new Makefile instructions to generate bundle operator using podman.
Changes RBAC files to make Monocle Operator work ClusterWide.
Add Operator bundle files.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -87,7 +87,7 @@ help: ## Display this help.
 
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	$(CONTROLLER_GEN) rbac:roleName=controller-manager crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
```
Comment (morucci) [inline Makefile:90]: in config/rbac/role.yaml I see that the role's name is 'manager-role' so is it ok to change this ?

File: Makefile [hunk 2]
```diff
@@ -47,7 +47,7 @@ ifeq ($(USE_IMAGE_DIGESTS), true)
 endif
 
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+IMG ?= controller:v0.13.0
```
Comment (morucci) [inline Makefile:50]: That should be the real image path from https://quay.io/organization/change-metrics. I'll build on add the image there.

File: config/rbac/role.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
   creationTimestamp: null
-  name: manager-role
```
Comment (morucci) [inline config/rbac/role.yaml:6]: Could you remind me why this change is needed ?

File: config/rbac/role.yaml [hunk 4]
```diff
@@ -1,33 +1,34 @@
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
+kind: ClusterRole
```
Comment (morucci) [inline config/rbac/role.yaml:3]: I expected we could make the operator work no cluster scoped...

File: Makefile (hunk 1 above)
Comment (fserucas) [reply Makefile]: changed

File: config/rbac/role.yaml (hunk 3 above)
Comment (fserucas) [reply config/rbac/role.yaml:6]: changed back.
